### ISSUE TITLE
Q3 2024: Creative content ideas that spark joy

### ISSUE THREAD

**Issue by jtemperton** [PostHog/meta#219]
## Summary

We want more great, creative ideas for content to run on PostHog dot com and also in the Product for Engineers newsletter. Great blogs, great newsletters, great blogs that become newsletters. More great, creative content also means more great social content. 

We're looking for ideas for more creative pieces of writing where we can take a strong point of view and defend it. These ideas should support PostHog's mission and point of view and, crucially, appeal to our core audience of founders and product engineers. Hot takes, but good takes. To help with this, we've created a weekly Ideas Cauldron meeting. 

Everyone who comes to this meeting should come along with at least one idea they want to work on and be ready to talk everyone through a quick pitch as to why it's a great idea and how they would write it. You can have more than one idea, that's great! You can also bring along ideas that you might not want to do but you feel that _someone_ should. We'll discuss ideas, yay/nay them, then crack on with research and writing.

Some personal faves that may or may not inspire you...

[Beyond the 10x engineer](https://newsletter.posthog.com/p/beyond-the-10x-engineer)
[The habits of effective remote teams](https://posthog.com/newsletter/remote-working) (yes, I'm biased)
[How first-time founders fail](https://posthog.com/newsletter/first-time-founders)
[Hiring (and managing) cracked engineers](https://posthog.com/founders/cracked-manifesto)

Some context, these newsletters have driven the **most subscriptions** on Substack since Product for Engineers launched:

![image](https://github.com/PostHog/posthog.com/assets/162187331/5be64dd7-2db3-43f3-a58a-6ca99029f33e)

My take on what works.

- How you, an engineer, can do things better
- How to succeed/not fail
- How we, an interesting and succesful company, do an interesting thing in an interesting way
- What we learned about X/Y/Z, and why it matters to you
- Smart takes on relatively buzzy things that we're good at: asynchronous working, remote working

And these have driven the least subscriptions:

![image](https://github.com/PostHog/posthog.com/assets/162187331/0e46f49b-ce66-4263-8931-f43dd53df1e2)

My take on what doesn't work? Things that are two naval-gazing and don't seem immediately actionable or relevant to our audience.

- What motivates me as a CEO
- What we learned about hiring from our first five employees
- The companies that shaped PostHog

## Goal

More great ideas that go from pitch to publication from a wide roster of writers. Wherever they're published, these pieces of content should be widely read and have strong engagement.

## Plan

- [x] Weekly ideas meeting
- [ ] Have content confirmed for the next three newsletters on a rolling basis throughout the quarter and beyond
- [ ] A regular flow of ideas so the marketing team has a good selection of content to work on

**Comment by andyvan-ph** [comment]
Hopefully this will come out of the meeting, but I think an additional output should be a specific ambitious writing project you're going to work on personally. Could be, but doesn't have to be, a newsletter thing.

**Comment by jtemperton** [comment]
Thanks again for all the ideas at the first meeting. Notes below broken down by who pitched what with some brief thoughts from me. I'm happy to take on a bunch of writing for projects without an obvious owner. Just let me know.

Next step would be to confirm which ideas we want to move forward with. If folks want to shout loud for the ideas they like best we can prioritize those.

@andyvan-ph 

- **The stages of product market fit.** Some good SEO benefit and a good chance to revisit PMF. Question before we definitely do this: what are the stages, and is there enough to say about them?

- **How to write a company handbook / Why you need to write a company handbook.** Good chance to get some extra love for our handbook and speak to other companies with great, transparent handbooks. This one feels like a no-brainer. I'd be happy to take it on if you like.

- **Important lessons from TKTK book here TKTK.** Basic idea here is we read a (very engineering-relevant) book so you don't have to, and then share key lessons from it. Suggestion was to start with The Mom Test.

- **The habits of effective engineers.** Perhaps a similar format to '[How first-time founders fail](https://newsletter.posthog.com/p/how-first-time-founders-fail)' to share some actionable tips and ideas.

- **What it's like to be a founding engineer.** As above, but this one needs more of a purpose to work. What do we really want to find out?

- **A newsletter version of [this.](https://posthog.com/founders/what-recruiters-see)** Sorry, I lost the thread of the conversation here. Agree something along these lines would make a really strong newsletter, let me know how/if I can help.

@ivanagas 

- **The ultimate guide to becoming a great growth engineer / Why all engineers should be growth engineers.** Be good to see a pitch on this one/ a few lines on what we'd include. But the idea feels good.

- **How and why you should ship to production.** Would detail our process here, the benefits, challenges, etc. Fits well with the 'speed' vibe. I'm into it!

- **No design/no product management by default.** Sorry, my notes on this one aren't so great but regardless we need a more concrete headline for this one.

- **Why engineers should do support.** Great idea! By all means work on an outline/pitch for folks to review.

@Lior539 

- **Interviews with people behind popular dev tools.** Suggestion was to start with the person behind NPM. Sorry, I missed their name? Could work as a semi-regular newsletter format but would be good to work out what other tools we'd want to feature. Kinda like 'The untold story of TKTK,' which is fun!

- **The worst outage you faced and how you fixed it.** Interviews with founders/engineers about major outages they fixed at early-stage companies. I think the interview idea above is stronger.

@jtemperton 

- **How to design a killer website.** I think we're really good at this. I think we could have strong opinions about this. If we can distill the how and why down into something actionable I think this could be really good fun.

- **How to design the perfect work trial** We were included in [this Lenny article](https://www.lennysnewsletter.com/p/adding-a-work-trial-to-your-interview) but we should now totally do our own thing. I'm happy to take this one on.

**Comment by andyvan-ph** [comment]
Really noticeable how many more views posts with "engineers" in the headline get on first send!

<img width="1186" alt="Screenshot 2024-07-04 at 13 00 37" src="https://github.com/PostHog/meta/assets/92976667/26a11394-4d47-420e-b966-b7c4e3a3afb0">

**Comment by andyvan-ph** [comment]
Ones I think we should do:
- The habits of effective engineers
- How to get a job at a startup
- Why engineers should do support
- The ultimate guide to becoming a great growth engineer
- Important lessons from TKTK book here TKTK (I'm thinking "What we learned from reading the TKTK")
- The one about NPM
- How to design the perfect work trial
- The stages of product market fit (put my name on this for some time)

I will also look at the next part of the growth series at some point, but not sure how / when atm.

> The habits of effective engineers. Perhaps a similar format to '[How first-time founders fail](https://newsletter.posthog.com/p/how-first-time-founders-fail)' to share some actionable tips and ideas.

@jtemperton you fancy picking this one up? I think if we did it in a way where we leaned into "we talked to XYZ people" and this is what they said, it could work super well. Great fit for the audience and lots of Hacker News potential. If we wanted to get specific, we could do it "what YC founders think" or what engineering-led startups think. 

> A newsletter version of [this.](https://posthog.com/founders/what-recruiters-see) Sorry, I lost the thread of the conversation here. Agree something along these lines would make a really strong newsletter, let me know how/if I can help.

I think this was more like "How to get a job at a startup", which could incorporate some of this post and others we done.

> The worst outage you faced and how you fixed it. Interviews with founders/engineers about major outages they fixed at early-stage companies. I think the interview idea above is stronger.

I wonder if we could / should do something that's like "How to deal with outages"? Could draw on the experiences of others, but also talk about communication, include some examples of famous good and bad ones.
